ebms-send-in/src/main/kotlin/no/nav/emottak/ebms/App.kt

```diff
@@ -46,9 +45,9 @@ fun main() {
     // database.migrate()
 
     System.setProperty("io.ktor.http.content.multipart.skipTempFile", "true")
-    embeddedServer(Netty, port = 8080, module = Application::ebmsSendInModule, configure = {
-        this.maxChunkSize = 100000
-    }).start(wait = true)
+    embeddedServer(Netty, port = 8080, module = Application::ebmsSendInModule)
+        .also { it.engineConfig.maxChunkSize = 100000 }
+        .start(wait = true)
```

Comment:
Jeg ser at den opprinnelige koden er lettere å lese og forstå. Kanskje vi kan vurdere å beholde den for å gjøre koden mer lesbar og vedlikeholdbar? :thought_balloon: